python/ray/tune/schedulers/async_hyperband.py

```diff
@@ -25,9 +25,11 @@ class AsyncHyperBandScheduler(FIFOScheduler):
             Note that you can pass in something non-temporal such as
             `training_iteration` as a measure of progress, the only requirement
             is that the attribute should increase monotonically.
-        reward_attr (str): The training result objective value attribute. As
+        metric (str): The training result objective value attribute. As
             with `time_attr`, this may refer to any objective value. Stopping
             procedures will use this attribute.
+        mode (str): One of {min, max}. Determines whether objective is minimizing
+            maximizing the metric attribute
```

Comment:
(this applies to all) - Can you also keep `reward_attr` here, but raise a DeprecationWarning if  set to something not None?